docs/Tutorials/BeginnersTutorial.md

```diff
@@ -187,7 +187,7 @@ depending on what version of `Paraview` you have.)
 \image html beginners_paraview_top.png "Paraview top-bar"
 
 Now you'll have a solid sphere with highlighted lines. To view the interior of
-the domain, you'll need to add a filter. Probably the two most helpful filters
+the domain, you'll need to add a filter. To access the filters, navigate to `Filters` on the top menu bar, hover over `Alphabetical`, and search for your filter of choice.  Probably the two most helpful filters
```

Comment:
This line is too long. For our formatting style, we keep lines 80 characters or less. You can either change this by hand, or , if you use VSCode, there are many extensions that will detect and auto-wrap lines longer than 80 characters.